libs/ledger-live-common/src/market/v2/useMarketDataProvider.ts

```diff
@@ -0,0 +1,128 @@
+import { useQueries, useQuery } from "@tanstack/react-query";
+import {
+  fetchCurrency,
+  fetchCurrencyChartData,
+  fetchCurrencyData,
+  fetchList,
+  getSupportedCoinsList,
+  supportedCounterCurrencies,
+} from "../api/api";
+import {
+  MarketCurrencyRequestParams,
+  MarketListRequestParams,
+  MarketListRequestResult,
+} from "../types";
+import { QUERY_KEY } from "./queryKeys";
+import { listSupportedCurrencies } from "../../currencies";
+import { CryptoCurrency } from "@ledgerhq/types-cryptoassets";
+import { useMemo } from "react";
+
+// 1min
+const REFETCH_TIME = 1 * 60000;
```

Comment:
I'd suggest we abstract all configuration away from business logic, potentially a separate constants core folder, defaulting on an env variable rather than a Feature Flag ... 

Unsure of infrastructure architecture/deployment times/abstraction but could we rather set the value (if available) against an env var ... 

There have been multiple issues with the caching of Firebase values in LL (sometimes being stale for 13 + hours), unsure if this is resolved, but semantically I'd presume Firebase config is related to Feature Flags and not Req configuration ...

Reply:
Yup you're absolutely right ! 

I'll try to put away all that config :) 

Currently this `REFETCH_TIME ` is just a fallback but we need to handle it more globally for sure